Change functionPerRoute to false by default (#8546)
withastro/astro: packages/integrations/vercel/README.md

### Function bundling configuration

The Vercel adapter combines all of your routes into a single function by default.

> You also have the option to split into a seperate function for each route using the `functionPerRoute` option. Doing so means your functions are smaller, you're less likely to exceed the size limit, and code starts are faster. 

Reviewer: ```suggestion
You also have the option to split builds into a separate function for each route using the `functionPerRoute` option. This reduces the size of each function, meaning you are less likely to exceed the size limit for an individual function. Also, code starts are faster. 
```
I added a bit because people will skim and confuse **size of a function** with "size = NUMBER of functions"... and they are totally at risk of exceeding THAT limit.

Author: ah yeah good point.

> The free tier of Vercel limits each deployment to no more than 12 functions. If you have a paid plan, using turning on `functionPerRoute` is something to consider enabling.

Reviewer: ```suggestion
Verify that your Vercel plan includes an appropriate number of functions before enabling `functionPerRoute`. For example, Vercel's free tier limits each deployment to no more than 12 functions. If your Vercel plan is insufficient for the number of routes in your project, you will receive an error message during deployment.
```
